MiniGamesBox Inventory/src/main/java/plugily/projects/minigamesbox/inventory/normal/FastInv.java

```diff
@@ -1,398 +0,0 @@
-import org.bukkit.event.inventory.InventoryType;
-import org.bukkit.inventory.Inventory;
-import org.bukkit.inventory.InventoryHolder;
-import org.bukkit.inventory.ItemStack;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.function.Consumer;
-import java.util.function.Predicate;
-import java.util.stream.IntStream;
-
-/**
- * Lightweight and easy-to-use inventory API for Bukkit plugins.
- * The project is on <a href="https://github.com/MrMicky-FR/FastInv">GitHub</a>.
- *
- * @author MrMicky
- * @version 3.0.3
- */
-public class FastInv implements InventoryHolder {
-
-  private final Map<Integer, Consumer<InventoryClickEvent>> itemHandlers = new HashMap<>();
-  private final List<Consumer<InventoryOpenEvent>> openHandlers = new ArrayList<>();
-  private final List<Consumer<InventoryCloseEvent>> closeHandlers = new ArrayList<>();
-  private final List<Consumer<InventoryClickEvent>> clickHandlers = new ArrayList<>();
-
-  private final Inventory inventory;
-
-  private Predicate<Player> closeFilter;
-
-  /**
-   * Create a new FastInv with a custom size.
-   *
-   * @param size The size of the inventory.
-   */
-  public FastInv(int size) {
-    this(size, InventoryType.CHEST.getDefaultTitle());
-  }
-
-  /**
-   * Create a new FastInv with a custom size and title.
-   *
-   * @param size  The size of the inventory.
-   * @param title The title (name) of the inventory.
-   */
-  public FastInv(int size, String title) {
-    this(size, InventoryType.CHEST, title);
-  }
-
-  /**
-   * Create a new FastInv with a custom type.
-   *
-   * @param type The type of the inventory.
-   */
-  public FastInv(InventoryType type) {
-    this(Objects.requireNonNull(type, "type"), type.getDefaultTitle());
-  }
-
-  /**
-   * Create a new FastInv with a custom type and title.
-   *
-   * @param type  The type of the inventory.
-   * @param title The title of the inventory.
-   */
-  public FastInv(InventoryType type, String title) {
-    this(0, Objects.requireNonNull(type, "type"), title);
-  }
-
-  private FastInv(int size, InventoryType type, String title) {
-
-    if(type == InventoryType.CHEST && size > 0) {
-      int invSize = getInventorySize(size);
-      this.inventory = Bukkit.createInventory(this, invSize, title);
-    } else {
-      this.inventory = Bukkit.createInventory(this, type, title);
-    }
-
-    if(this.inventory.getHolder() != this) {
-      throw new IllegalStateException("Inventory holder is not FastInv, found: " + this.inventory.getHolder());
-    }
-  }
-
-
-  private int getInventorySize(int size) {
-    if(size <= 9) {
-      return 9;
-    }
-    if(size <= 9 * 2) {
-      return 9 * 2;
-    }
-    if(size <= 9 * 3) {
-      return 9 * 3;
-    }
-    if(size <= 9 * 4) {
-      return 9 * 4;
-    }
-    if(size <= 9 * 5) {
-      return 9 * 5;
-    }
-    if(size <= 9 * 6) {
-      return 9 * 6;
-    }
-    return 9;
-  }
```

Comment:
inv size check missing, may should be added in general to not produce wrong size invs that could break the lib because the size is unchecked